server/test/lib/scene/scene.duplicate.test.js

```diff
@@ -0,0 +1,24 @@
+const { expect, assert } = require('chai');
+const EventEmitter = require('events');
+const SceneManager = require('../../../lib/scene');
+
+const event = new EventEmitter();
+
+describe('SceneManager', () => {
+  it('should duplicate a scene', async () => {
+    const sceneManager = new SceneManager({}, event);
+
+    const duplicatedScene = await sceneManager.duplicate('to-duplicate-scene', 'new-name', 'new-icon');
+    expect(sceneManager.scenes['new-name']).not.to.be.undefined; // eslint-disable-line
```

Comment:
To avoid having to disable eslint for this, you can do `.not.to.equal(undefined)` for example